### PR TITLE
Claims of integer and double types implemented

### PR DESCRIPTION
**Pull request by sa2304**
Service that we integrated with require 'date' claim to be a number. Since Json::Value supports integer & double types I just added api for them.

### CODE REVIEW

File: include/export/jwtpp/jwtpp.hh [hunk 1]
```diff
@@ -218,6 +218,9 @@ private:
 
 /**
 * \brief
+*
+* TODO Make template class to allow set any claims value of type supported by Json::Value.
```
Comment (troian) [inline include/export/jwtpp/jwtpp.hh:222]: mind make issue and reference it here?

File: include/export/jwtpp/jwtpp.hh [hunk 2]
```diff
@@ -245,6 +248,13 @@ private:
 			std::string s = _claims->operator[](key).asString();
 			return s == value;
 		}
+		
+		bool any(const std::string &key, Json::UInt value) { return _claims->operator[](key).asUInt() == value; }
+        bool any(const std::string &key, Json::Int value) { return _claims->operator[](key).asInt() == value; }
```
Comment (troian) [inline include/export/jwtpp/jwtpp.hh:253]: tabs for indents please

File: include/export/jwtpp/jwtpp.hh [hunk 3]
```diff
@@ -280,6 +290,31 @@ private:
 		std::string any(const std::string &key) {
 			return _claims->operator[](key).asString();
 		}
+		
+        Json::Int anyInt(const std::string &key) {
```
Comment (troian) [inline include/export/jwtpp/jwtpp.hh:294]: tabs

File: include/export/jwtpp/jwtpp.hh [hunk 4]
```diff
@@ -295,7 +330,13 @@ private:
 	public:
 		explicit set(Json::Value *c) : _claims(c) {}
 	public:
+        void any(const std::string &key, Json::UInt value) { _claims->operator[](key) = value; }
```
Comment (troian) [inline include/export/jwtpp/jwtpp.hh:333]: tabs

**Comment by sa2304** [comment]
Changed spaces to tabs & created the issue.
https://github.com/troian/jwtpp/issues/38

**Pull request merged**